conf/spec.yaml

```diff
@@ -325,6 +325,18 @@ package:
         500:
           handler: html_package_service_allow_error
           message: 'Allowed package installation error - %{title}: %{desc} %{reason}'
+    uninstall:
```

Comment:
Please also update CHANGELOG with the note for next release (1.0.17) and also add sample on the README.